### DISCUSSION TITLE
How do we utilize watchQuery hooks

### DISCUSSION THREAD

**Discussion by Ripul01** [heftapp/graphql_codegen#298] [Q&A]
I want to use the Watch Query hook to get data from my server. The problem is I don't quite understand how to use it. I tried using the stream property inside the streambuilder but it doesn't seem to rebuild on connection state changes. Please guide me if you've used it before. Thanks.

**Comment by budde377** [comment]
I think this question is better answered on the GitHub repo for `graphql` and `graphql_flutter`.

**Reply by Ripul01** [reply]
I see.  Thanks anyway.
